Jellyfin.Data/Entities/ProviderMapping.cs

```diff
@@ -43,12 +43,6 @@ public ProviderMapping(string providername, string providersecrets, string provi
             if (string.IsNullOrEmpty(providerdata)) throw new ArgumentNullException(nameof(providerdata));
             this.ProviderData = providerdata;
 
-            if (_user0 == null) throw new ArgumentNullException(nameof(_user0));
-            _user0.ProviderMappings.Add(this);
-
-            if (_group1 == null) throw new ArgumentNullException(nameof(_group1));
-            _group1.ProviderMappings.Add(this);
-
```

Comment:
If `_user0` and `_group1` parameters are removed, can they also be removed from the `Create` function?

Reply:
I removed those purely to stop compiler errors, this part of the schema is unused as of now